pkg/controller/scheduledscaling/scheduled_scaling.go

```diff
@@ -266,13 +266,15 @@ func (c *Controller) adjustHPAScaling(ctx context.Context, hpa *autoscalingv2.Ho
 		return nil
 	}
 
-	highestExpected, highestObject := highestActiveSchedule(hpa, activeSchedules)
+	highestExpected, usageRatio, highestObject := highestActiveSchedule(hpa, activeSchedules, current)
 
 	highestExpected = int64(math.Min(float64(highestExpected), float64(hpa.Spec.MaxReplicas)))
 
+	fmt.Println(highestExpected, current)
```

Comment:
Lets remove this print, or change it to logging.

Reply:
Thanks! Fixed!